fili-core/src/main/java/com/yahoo/bard/webservice/async/ResponseException.java

```diff
@@ -88,14 +152,22 @@ private static String buildMessage(
             String description,
             int statusCode,
             DruidQuery<?> druidQuery,
-            Throwable cause
+            Throwable cause,
+            ObjectWriter objectWriter
     ) {
+        String druidQueryString;
+        try {
+            druidQueryString = objectWriter.writeValueAsString(druidQuery);
+        } catch (JsonProcessingException e) {
+            druidQueryString = druidQuery.toString();
+            LOG.warn(String.format("Failed to properly serialize druid query %s", druidQueryString), e);
```

Comment:
If I do that, then the best we can say is "Failed to properly serialize a druid query." Which admittedly isn't going to be that different from what will be displayed now, since we don't override `toString`.